workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -104,22 +96,50 @@ describe('DynamicAuthorizationService', () => {
       ).rejects.toThrow(Error);
     });
   });
+
   describe('addUserToGroup', () => {
-    it('returns userID and groupID when user was successfully added to group', async () => {
-      mockGroupManagementPlugin.addUserToGroup = jest
-        .fn()
-        .mockResolvedValue({ data: { userId: 'userId', groupId: 'groupId' } });
+    it('returns userID and groupID when user was successfully added to the group', async () => {
+      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue({ data: { userId, groupId } });
 
-      const request = {
-        groupId: 'groupId',
-        userId: 'userId',
+      const { data } = await dynamicAuthzService.addUserToGroup({
+        groupId,
+        userId,
         authenticatedUser: mockUser
-      };
+      });
+
+      expect(data).toStrictEqual({ userId, groupId });
+    });
 
-      const { data } = await dynamicAuthzService.addUserToGroup(request);
+    it('throws when the user cannot be added', async () => {
+      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockRejectedValue(new Error());
```

Comment:
be more specific on what type of error is being thrown

Reply:
This just testing that the service passes along whatever error the plugin throws, so a specific error isnt required

Reply:
I would try to throw the specific error that mockManagementPlugin can throw, not just some generic error